src/surface/flight_control/flight_control/manual_control_node.py

```diff
@@ -49,6 +49,9 @@
 
 CONTROLLER_MODE_PARAM = 'controller_mode'
 
+NEXT_INSTR_FRAC = 0.05
```

Comment:
:Final for type safety

Reply:
Should we do this in this one place, even though we don't do it anywhere else?

Reply:
You have to start somewhere.